dts/arm/nxp/nxp_rt10xx.dtsi

```diff
@@ -1160,3 +1160,5 @@
 	 */
 	status = "disabled";
 };
+
+dvp_fpc24_interface: &csi {};
```

Comment:
Probably want to put this at the board level- what this line effectively says is "the CSI is broken out to a 24 pin dvp FPC interface" which is only true for the RT10xx EVK boards, not for all RT10xx SOCs

Reply:
Okay ... so nxp_rt10xx is for all RT10xx SoCs not only EVK boards ... so there is not a "common" place to put common things for RT10xx (also RT11xx) EVKs / EVKB series ? As we know that CSI is a common IP for those boards, it is a bit "inconvenient" to duplicate code for each board ? But well, I will do as your suggestion then.

Reply:
No, there isn't unfortunately- although we do duplicate a lot across our RT10xx EVKs, Zephyr does not have any infrastructure I'm aware of for sharing a DTS file describing common board properties across multiple boards